Makefile

```diff
@@ -47,14 +47,29 @@ GO111MODULE = on
 # ====================================================================================
 # Setup Kubernetes tools
 
+UP_VERSION = v0.13.0
+UP_CHANNEL = stable
 -include build/makelib/k8s_tools.mk
 
 # ====================================================================================
 # Setup Images
 
-DOCKER_REGISTRY ?= crossplane
-IMAGES = provider-jet-equinix provider-jet-equinix-controller
--include build/makelib/image.mk
+IMAGES = provider-jet-equinix
+-include build/makelib/imagelight.mk
+
+# ====================================================================================
+# Setup XPKG
+
+XPKG_REG_ORGS ?= xpkg.upbound.io/crossplane index.docker.io/crossplane
```

Comment:
If you want to publish in equinix org on marketplace we would need to update to `xpkg.upbound.io/equinix` -- is that preferable?